yew/src/html/classes.rs

```diff
@@ -0,0 +1,647 @@
+use indexmap::IndexSet;
+use std::{
+    borrow::{Borrow, Cow},
+    iter::FromIterator,
+};
+
+/// A set of classes.
```

Comment:
Fixed in c91407d

I couldn't find a way to make it clickable. I tried:

```
[`classes!`]
[`crate::classes!`]
[`self::classes!`]
[`yewclasses!`]
```

Nothing worked

Reply:
Ah, you can't add the exclamation mark (for technical reasons). I would write it like this: [\`classes!`][yew::classes]
That way it still shows up as `classes!`.

*Oh and just to make sure, intra doc links still require nightly Rustdoc for the moment (I think). The feature was stabilized a few weeks ago but no new version has been released yet.*

Reply:
Ah never mind, I was wrong. Linking to `yew::classes!` works on my end. Might be that you aren't using the nightly version locally?

Either way, I'd still write it as: [`classes!][yew::classes!] (can't use just [\`classes!`] because it's not in scope)

Reply:
Ah yes I don't use nightly. Ok it's fixed and working (in nightly) 76db907